test/wasm2c/export-names.txt

```diff
@@ -844,8 +845,9 @@ static void init_instance_import(w2c_test* instance, struct w2c_0x5Cmodule* w2c_
   instance->w2c_0x5Cmodule_import0x200x2A0x2F = w2c_0x5Cmodule_import0x200x2A0x2F(w2c_0x5Cmodule_instance);
 }
 
-const u32 wasm2c_test_min_0x5Cmodule_import0x200x2A0x2F = 0;
-const u32 wasm2c_test_max_0x5Cmodule_import0x200x2A0x2F = 4294967295;
+const u64 wasm2c_test_min_0x5Cmodule_import0x200x2A0x2F = 0;
+const u64 wasm2c_test_max_0x5Cmodule_import0x200x2A0x2F = 65536;
+const u8 wasm2c_test_is64_0x5Cmodule_import0x200x2A0x2F = 0;
```

Comment:
Would it make more sence for `_is64` , `_min` and `_max` to be suffixes?

Or perhaps we should define a wasm_rt_limits struct that can contain all three of these?   I think this could be a followup since this change a clearly incremental on what we have already.

Reply:
Makes sense. We can track this in a separate bug I think